src/analysis/utils/app-utils.ts

```diff
@@ -84,24 +84,29 @@ export const getAppStatusForDisplay = (status: LeoAppStatus): DisplayAppStatus =
     ['prestarting', () => 'Resuming'],
     ['prestopping', () => 'Pausing'],
     ['provisioning', () => 'Creating'],
-    [Utils.DEFAULT, () => _.capitalize(status)]
+    // TODO: is this case valid?
```

Comment:
Yes, for example 'running' falls into this case.

Reply:
Oops, this comment was mean to be "cast", not "case". It seems like type safety could be improved here by using a `Record<LeoAppStatus, DisplayAppStatus>` and listing all possible values.

Reply:
Updated this comment in https://github.com/DataBiosphere/terra-ui/pull/4154/commits/56adec16706736fd19bc90f3997035c3b9d5c61c.